Assets/Scripts/Gameplay/Mecha/MechaController.cs

```diff
@@ -168,6 +216,7 @@ protected override void OnEnable()
                 _rigidbody = GetComponent<Rigidbody>();
             EventManager.AddListener(Constants.TypedEvents.Inputs.OnLookAround, OnLookAround);
             EventManager.AddListener("OnMove", OnMove);
+            EventManager.AddListener("OnDash", OnDash);
```

Comment:
If you have some time to create a constant in [Constants.cs](https://github.com/Taliayaya/Project-86/blob/main/Assets/Scripts/Constants.cs) instead of hardcoding the string, it could be great